docs/learn-scams.md

```diff
@@ -0,0 +1,37 @@
+---
+id: learn-scams
+title: How to Protect Yourself from Scams
+sidebar_label: How to Protect Yourself from Scams
+---
+
+Scams have unfortunately made an impact on our community. It's important to stay alert and protect
+yourself and your non-refundable crypto assets from scammers.
+
+### Types of Scams
+
+- Personal messages over Telegram, and other social medias
+- 'Send us DOT/KSM, we'll send you double'
+- Emails asking for DOT/KSM addresses, posing as a member of our team
+
+### Admin will never contact you over Telegram
+
+If you've received a message from an admin over Telegram, ignore it. Our Polkadot team will never
+personally message you. Our social media's are posted on our website and any new social media's will
+be announced by our team.
+
+### Always check the source
+
+For any potential scam, always be sure to do a background check on the source, i.e, look at any
+username, email, Youtube channel name, etc. If something seems fishy, that's because it likely is.
+
+### Check twice before sending DOT/KSM
+
+A good practice to take in consideration is to verify the address you are sending crypto to. If you
+don't know that account, you probably shouldn't be sending your assets to there. It is very
+important to understand that crypto is a decentralized space, if you lose your assets, that is on
```

Comment:
Does it sound too harsh here if I bluntly say 'that is on you' ?

Reply:
perhaps it'd sound better to say 'it is your loss' ?

Reply:
I would rephrase to say something like "It's your responsibility to make sure that you understand where you are sending your funds to, crypto is a decentralized space and your only recourse if a mistake is made is by appealing to the council (who usually will not get involved in matters of mistaken transfers)."